src-java/connecteddevices-topology/connecteddevices-storm-topology/src/main/resources/connecteddevices-topology.yaml.example

```diff
@@ -0,0 +1,16 @@
+# topology configuration
```

Comment:
I would like to not have these *-topology.yaml.exmapl files. There are too many of them. We have issues with supporting ... much lover count of example config for existing components. We will fail to support these examples.

Reply:
Make sense. Removed.